app/utilities/schema.py

```diff
@@ -98,14 +98,15 @@ def get_allowed_languages(schema_name, launch_language):
     return [DEFAULT_LANGUAGE_CODE]
 
 
-def load_schema_from_metadata(metadata):
+def load_schema_from_metadata(*, metadata, language_code=None):
```

Comment:
We should enforce the keyword argument just for `language_code`. Normally the keyword argument isn't enforced for the thing the method is named after, i.e `load_schema_from_metadata` implies you need to pass in metadata. Plus if you only had 2 args, enforcing on 1 covers all cases.

Reply:
Only enforcing language code now 👍